english/friend-scream/follow.md

```diff
@@ -0,0 +1 @@
+You followed the sound to the ground floor and you realize, the noise is just another recording on a loop. WTF! You placed yourself back in danger for nothing. Where did they get that recording from? Who are 'they'? These are all questions you ask yourself. Just then, you see a figure in the distance. It's a...It's a...another Marshmallow man! This time he's more than just hands. He is HUGE! and he has sharp teeth! you realize that you've noticed these things about him because he is now 10 feet away from you! What do you do?!....
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.